utils/hct/pkgesbuild/status-email-template.html

```diff
@@ -0,0 +1,20 @@
+<style>
+table, th, td { border:1px solid black; border-collapse:collapse; font-family:Calibri; }
+th, td { padding-left: 10px; padding-right: 20px;  padding-top: 4px;  padding-bottom: 4px; }
+h3, body { font-family:Calibri; }
+</style>
+<p>
+{status_tables}
+</p>
+<p>
+Build Definition: $(BUILD_DEFINITIONNAME)<br/>
```

Comment:
So, build ES runs your powershell, which outputs this file with {status_tables} filled in, and then it does string replacement on the $(VARIABLES)?

Reply:
Yep.